lib/rubocop/options.rb

```diff
@@ -34,7 +34,7 @@ def parse(command_line_args)
     private
 
     def args_from_file
-      if File.exist?('.rubocop')
+      if File.file?('.rubocop')
```

Comment:
Or, maybe use [`File.readable?`](https://ruby-doc.org/core-2.2.0/File.html#method-c-readable-3F) instead of `File.file?`.